cookies_test.go

```diff
@@ -40,6 +40,18 @@ func TestCookies_Set(t *testing.T) {
 	r.Equal("name=\"Rob Pike\"; Max-Age=86400", h)
 }
 
+func TestCookies_SetWithPath(t *testing.T) {
+	r := require.New(t)
+	res := httptest.NewRecorder()
+
+	c := Cookies{&http.Request{}, res}
+
+	c.SetWithPath("name", "Rob Pike", "/foo")
+
+	h := res.Header().Get("Set-Cookie")
+	r.Equal("name=\"Rob Pike\"; Path=/foo", h)
+}
+
```

Comment:
Shouldn't you also test with the "/" path to ensure that this cookie is not set for that?